src/argilla/server/models/models.py

```diff
@@ -60,7 +61,7 @@ class Field(DatabaseModel):
     name: Mapped[str] = mapped_column(Text, index=True)
     title: Mapped[str] = mapped_column(Text)
     required: Mapped[bool] = mapped_column(default=False)
-    settings: Mapped[dict] = mapped_column(JSON, default={})
+    settings: Mapped[dict] = mapped_column(MutableDict.as_mutable(JSON), default={})
```

Comment:
Do we need this line? Or just prevent a prover config for future event handlers listening to the `onChanged` event?